cfme/common/provider.py

```diff
@@ -131,17 +137,7 @@ def version(self):
 
     @property
     def category(self):
-        # Prevent circular imports
-        from utils.providers import (
-            cloud_provider_type_map, infra_provider_type_map, container_provider_type_map)
-        if self.type in cloud_provider_type_map:
-            return "cloud"
-        elif self.type in infra_provider_type_map:
-            return "infra"
-        elif self.type in container_provider_type_map:
-            return "container"
-        else:
-            return "unknown"
+        return self.type_tclass
```

Comment:
could we just use the attribute category directly, or perhaps remove it in favour of the new name